_posts/2017-05-26-deploying-create-react-app-to-s3.markdown

```diff
@@ -0,0 +1,267 @@
+---
+layout: post
+title:  "Deploying create-react-app via S3"
+date:   2017-06-01 00:00:00 -0000
+categories: react devops
+author: Zach Cotter
+---
+
+At Forward Financing, we've started using
+[create-react-app][create-react-app]
+as a starter kit for all of our new React projects. Create-React-App is awesome
+because it comes with a one size fits all webpack configuration via the
+`react-scripts` package. All you have to do is `npm run build` and you have a
+ready to ship `/build` folder with minified production asset files.
+
+The first project we did this with was a front-end for an application used internally to manage our user's roles and access to pages across our suite
+of apps. The backend server was our standalone single sign on service,
+written in Elixir+Phoenix, which had some existing server rendered views. An
```

Comment:
~~which had some existing server rendered views.~~

extra context that might not be relevant to the topic. we can tighten it up by removing it.